server/build.gradle.kts

```diff
@@ -70,6 +70,9 @@ dependencies {
     // uncomment to test extensions directly
 //    implementation(fileTree("lib/"))
     implementation(kotlin("script-runtime"))
+
+    testImplementation("io.mockk:mockk:1.9.3")
+    testImplementation("io.javalin:javalin:4.0.0")
```

Comment:
Just wondering, how does this help?

Reply:
Tested the controller. Needed them to compile :)

Reply:
for this more explicitly
```kotlin
    @Test
    fun `POST non existent Category Id should give error`() {
        every { ctx.formParam("category") } returns "1"
        UpdateController.categoryUpdate(ctx)
        verify { ctx.status(HttpCode.BAD_REQUEST) }
        val updater by DI.global.instance<IUpdater>()
        assertEquals(0, updater.getStatus().value.numberOfJobs)
    }
 ```

Reply:
Weird, tests should use regular dependencies + test dependencies, not sure why it needs javalin

Reply:
Just tried a `:server:clean :server:test` and it works without it. So i guess i just had a weird run back then.

Reply:
👍